scripts/realsense-metadata-jammy-master.patch

```diff
@@ -1,4 +1,4 @@
-From 68ebc4e3fb3729f2b6804ea4d72d5dfda356512f Mon Sep 17 00:00:00 2001
+From f8c4dca6a001d42a79a2a7a5d9fc4748bb5df25c Mon Sep 17 00:00:00 2001
```

Comment:
Will this handle 6.8?

Reply:
no

Reply:
So which kernel version is handled on *-master.patch?